UI/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<meta charset="UTF-8"/>
+		<title>Моя перва HTML страница</title>
+		<link href = "style.css" rel = "stylesheet" type = "text/css"/>
+	</head>
+	<body>
+		<div class = "Wallpaper"></div>
+		
+		
+		<div class = "Header"></div>
+		<div class = "HeaderIcon">HUKUMKA</div>
+		<div class = "HeaderUserName">username</div>
+		<div class = "HeaderRectangle"></div>
+		<a href = "https://vk.com/justakel" class = "HeaderLogin">sign out</a>
+		
+		
+		<div class = "Footer"></div>
+		<div class = "FooterIcon">HUKUMKA</div>
+		<div class = "FooterData">01.03.2021</div>
+		<div class = "FooterName">Hukumka Inc.</div>
+		<div class = "FooterInfo">2 course, 8 group</div>
+		<div class = "FooterEmail">nikitakondibor@gmail.com</div>
+		
+		
+		<div class = "Filter"></div>
+		<div class = "FilterName">filter by</div>
+		<div class = "FilterSaleRectangle"></div>
+		<div class = "FilterSale">sale</div>
+		<div class = "FilterDataRectangle"></div>
+		<div class = "FilterData">data</div>
+		<div class = "FilterInfoNameRectangle"></div>
+		<div class = "FilterInfoName">name</div>
+		<a href = "https://vk.com/hanna_varhan" class = "FilterFind">find</a>
+		
+		
+		<div class = "Window1"></div>
+		<div class = "WindowEstimate">kokoko</div>
+		<div class = "WindowTag">#tag1 #hukumka #tag2 #sale</div>
+		<div class = "WindowPicture"></div>
+		<div class = "WindowName">Name</div>
+		<input type = "text" class = "WindowDescription">
+		<div class  = "WindowStars">Возможно, скоро будут звездочки</div>
+		<div class = "WindowVendor">vendor</div>
+		<div class = "WindowLearnMore">Learn more...</div>
+		<div class = "WindowData">period(data)</div>
+		<div class = "WindowComment1">знаю, что не очень, но зато сам</div>
+		<div class = "WindowComment2">не судите строго, я старался</div>
+		<div class = "WindowPercent">99%</div>
+		
+		<div class = "Window2"></div>
+		<div class = "WindowTag2">#tag1 #hukumka #tag2 #sale</div>
+		<div class = "WindowPicture2"></div>
+		<div class = "WindowName2">Name</div>
+		<input type = "text2" class = "WindowDescription2">
+		<div class  = "WindowStars2">Возможно, скоро будут звездочки</div>
+		<div class = "WindowVendor2">vendor</div>
+		<div class = "WindowLearnMore2">Learn more...</div>
+		<div class = "WindowData2">period(data)</div>
+		<div class = "WindowComment12">не очень понял, как скомпоновать все окошки</div>
+		<div class = "WindowComment22">поэтому не стал заполнять ручками остальные окошки</div>
+		<div class = "WindowPercent2">99%</div>
+		<div class = "WindowEstimate2"></div>
+		
+		<div class = "Window3"></div>
```

Comment:
It would be easy to fill all the 'window'-s if you used correct structure:
```
<div class = "ad">
	<div class="ad-tags">#tag1 #hukumka #tag2 #sale</div>
	<div class = "ad-photo"></div>
	<h3 class="ad-title">Name</h3>
	<p class="ad-description">Description</p> <!--- input tags are used to allow user to enter some data, here you need just to show the description --->
	<div class  = "ad-rating">
               <span class="ad-rating-star filled">*</span>
               <span class="ad-rating-star filled">*</span>
               <span class="ad-rating-star filled">*</span> <!---second class to indicate the average rating of ad --->
               <span class="ad-rating-star">*</span>
               <span class="ad-rating-star">*</span>
        </div>
	<div class="ad-vendor">vendor</div>
	<a class="ad-learn-more-link" href="#">Learn more...</a> <!--- a as for links --->
	<p class="ad-period">period(data)</p>
        <div class="ad-comments">
		<div class="ad-comment">не очень понял, как скомпоновать все окошки</div>
		<div class="ad-comment">поэтому не стал заполнять ручками остальные окошки</div>
        </div>
</div>
````